projects/storefrontlib/src/lib/product/components/product-list/product-facet-navigation/product-facet-navigation.component.spec.ts

```diff
@@ -1,16 +1,184 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { ProductFacetNavigationComponent } from './product-facet-navigation.component';
 import { NgbCollapseModule, NgbModalModule } from '@ng-bootstrap/ng-bootstrap';
+import { DebugElement, ChangeDetectionStrategy } from '@angular/core';
+import { By } from '@angular/platform-browser';
 
 describe('ProductFacetNavigationComponent in product-list', () => {
   let component: ProductFacetNavigationComponent;
   let fixture: ComponentFixture<ProductFacetNavigationComponent>;
+  let element: DebugElement;
+
+  const mockFacets = [
```

Comment:
I would also reduce this mock a little. No need for that many values for each facet.